### PR TITLE
[vm-repair] Bug fix 

### PR DESCRIPTION
**Pull request by haagha**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/vm-repair/azext_vm_repair/_validators.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def validate_create(cmd, namespace):
         namespace.repair_vm_name = ('repair-' + namespace.vm_name)[:14] + '_'
 
     # Check copy disk name
-    timestamp = datetime.utcnow().strftime('%Y%m%d%H%M%S.%f')
+    timestamp = datetime.utcnow().strftime('%Y%m%d%H%M%S')
```
Comment (zhoxing-ms) [inline src/vm-repair/azext_vm_repair/_validators.py:48]: Please write the issue you fixed in history notes

**Comment by zhoxing-ms** [comment]
By the way, could you add some tests for this case?

**Comment by haagha** [comment]
@zhoxing-ms it's only a naming change for copy disk. The behaviour is validated with current tests that create copy disks.

File: src/vm-repair/azext_vm_repair/_validators.py (hunk 1 above)
Comment (haagha) [reply src/vm-repair/azext_vm_repair/_validators.py:48]: added history, thanks.

File: src/vm-repair/HISTORY.rst [hunk 2]
```diff
@@ -1,6 +1,9 @@
 
 Release History
 ===============
+0.4.0
+++++++
+Fixing issue in disk copy, removing floating point in disk name.
```
Comment (zhoxing-ms) [inline src/vm-repair/HISTORY.rst:6]:
```suggestion

0.4.0
++++++
Fixing issue in disk copy, removing floating point in disk name.
```

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**